tsl/src/nodes/decompress_chunk/planner.c

```diff
@@ -145,6 +146,26 @@ typedef struct
 
 } DecompressionMapContext;
 
+typedef struct VectorQualInfoDecompressChunk
```

Comment:
Let's merge these interface changes separately? I don't understand what's happening in TSBench, just to narrow it down.